### PR TITLE
Unify with other E+ Python Apps

### PR DESCRIPTION
**Pull request by Myoldmopar**

### CODE REVIEW

File: .github/workflows/flake8.yml [hunk 1]
```diff
@@ -18,4 +18,4 @@ jobs:
       run: pip install flake8
 
     - name: Run Flake8
-      run: flake8 eda
+      run: flake8 energyplus_diffs
```
Comment (Myoldmopar) [inline .github/workflows/flake8.yml:21]: Most of the changes are due to the source folder changing names.

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -22,5 +22,5 @@ jobs:
         run: pip install click matplotlib pandas nose coverage
 
       - name: Run Tests
-        run: nosetests --with-coverage --cover-package scp  # --cover-html for local html results
+        run: nosetests --with-coverage --cover-package energyplus_diffs  # --cover-html for local html results
```
Comment (Myoldmopar) [inline .github/workflows/tests.yml:25]: This was presumably not working

File: Make_Plots.ipynb [hunk 3]
```diff
@@ -11,8 +11,8 @@
     "\n",
     "import matplotlib.pyplot as plt\n",
     "\n",
-    "from eda.compress import make_archive\n",
-    "from eda.plot_vars import plot"
+    "from energyplus_diffs.compress import make_archive\n",
+    "from energyplus_diffs.plot_vars import plot"
```
Comment (Myoldmopar) [inline Make_Plots.ipynb:15]: Should "just work" in the notebook I suppose, right?

File: energyplus_diffs/__init__.py [hunk 4]
```diff
@@ -0,0 +1 @@
+VERSION = "0.2"
```
Comment (Myoldmopar) [inline energyplus_diffs/__init__.py:1]: Bumped version to 0.2

File: energyplus_diffs/__main__.py [hunk 5]
```diff
@@ -0,0 +1,4 @@
+from energyplus_diffs.cli import cli
+
+
+cli()
```
Comment (Myoldmopar) [inline energyplus_diffs/__main__.py:4]: New `python -m energyplus_diffs` entry point is just calling `cli()`, which is the exact same as pip installing the library and running the `energyplus_diffs` binary shim.  But some people like this better.  So here we are.

**Review by Myoldmopar — COMMENTED** [review]
Alright, pretty simple change here really, bump the version, add a __main__, and then rename the source folder and fix the issues.  Everything seems happy locally.

**Comment by Myoldmopar** [comment]
@mitchute let me know if you see any issues here, or if you can test it out in a notebook environment to make sure it's good to go.  Thanks!

**Comment by mitchute** [comment]
Looks good to me. Do we need to pin the dependencies to `>=` the current versions?

**Comment by Myoldmopar** [comment]
That's totally fine.  If you feel like adding it onto this set of changes, feel free, or send me the versions you are using and I'll pin them there since I already have this branch checked out and such.

**Comment by mitchute** [comment]
@Myoldmopar done. Looks good to me. Merge away when ready.

**Pull request merged**